src/libraries/tick.cairo

```diff
@@ -28,6 +28,7 @@ struct Info {
 
 #[starknet::interface]
 trait ITick<TStorage> {
+    fn tick_spacing_to_max_liquidity_per_tick(ref self: TStorage, tick_spacing: i32) -> u128;
```

Comment:
because we are using `ref self: TStorage` ?

Reply:
I think what @dpinones means is that you are not modifying the state so it should just be a snapshot

Reply:
true, lemme change this.

Reply:
> because we are using `ref self: TStorage` ?

This field is mandatory, but as @rcatalan98 points out, it is true that we can use a snapshot to indicate that the state is not being modified